generator/lookml.py

```diff
@@ -0,0 +1,73 @@
+"""Generate lookml from namespaces."""
+from itertools import groupby
+from pathlib import Path
+from typing import Dict, List
+
+import click
+import yaml
+from google.cloud import bigquery
+from jinja2 import Environment, FileSystemLoader
+
+_env = Environment(
+    loader=FileSystemLoader(
+        (Path(__file__).parent / "templates").absolute().as_posix()
+    ),
+    trim_blocks=True,
+    lstrip_blocks=True,
+)
+
+VIEW_TEMPLATE = _env.get_template("view.lkml")
+
+
+def _get_dimensions(schema: List[bigquery.SchemaField]):
+    for field in schema:
+        if field.mode == "REPEATED":
+            pass  # ignore arrays for now
+        elif field.field_type == "RECORD":
+            for name, field_type in _get_dimensions(field.fields):
+                yield f"{field.name}.{name}", field_type
+        else:
+            yield field.name, field.field_type
+
+
+def _get_lookml(client: bigquery.Client, view: str, tables: List[Dict[str, str]]):
+    dimensions = sorted(
+        {
+            (name, field_type)
+            for item in tables
+            for name, field_type in _get_dimensions(
+                client.get_table(item["table"]).schema
+            )
+        }
+    )
+    for name, group in groupby(dimensions, lambda x: x[0]):
+        assert 1 == len(list(group)), f"{name} has multiple types in tables for {view}"
+    return VIEW_TEMPLATE.render(
+        view=view, table=tables[0]["table"], dimensions=dimensions
```

Comment:
This currently generates a single view, with `sql_table_name: {{ table }}`, and needs to be modified to correctly handle multiple tables.